setup.py

```diff
@@ -41,11 +42,17 @@
         "Development Status :: 4 - Beta",
         "Environment :: Web Environment",
         "Framework :: Zope3",
+        "Framework :: Plone",
```

Comment:
This not a dedicated Plone add-on. We use it for plain zope 3 apps as well.

Reply:
Ah, we moved this to the collective. But still, Zope would be more appropriate I think.

Reply:
Ok, i have removed the Plone classifiers.